error: could not compile `eventuals`

# Actual:
`no `select` in the root`

# Expected:
Dependency should build

# Notes:
```
error[E0432]: unresolved import `tokio::select`
 --> /home/richardb/.cargo/registry/src/github.com-1ecc6299db9ec823/eventuals-0.4.0/src/eventual/eventual.rs:9:5
  |
9 | use tokio::select;
  |     ^^^^^^^^^^^^^ no `select` in the root
```

# Background
- crate version 0.4.0
- cargo 1.52.0 (69767412a 2021-04-21)
- rustc 1.52.1 (9bc8c42bb 2021-05-09)
- edition 2018

# Dependencies:
```
actix-web = { version = "3.3.2", features = ["rustls"] }
actix-cors = "0.5"
chrono = { version = "0.4", features = ["serde"] }
clap = "2.33.3"
eventuals = "0.4.0"
fern = "0.6"
futures-util = "0.3.15"
jsonwebtoken = "7.2.0"
log = "0.4"
rustls = "0.19.1"
serde = "1.0.126"
serde_json = "1.0.64"
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,11 +23,10 @@ maintenance = { status = "experimental" }
 
 [dependencies]
 by_address = "1.0"
-tokio = { version="1.8" }
+tokio = { version="1.8", features=["macros", "time", "rt", "sync"] }
 futures = "0.3.15"
 never = "0.1.0"
 
 [dev-dependencies]
-tokio = { version="1.8", features=["full"] }
 eventuals = { path=".", features=["trace"] }
 lazy_static = "1.0"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "eventuals"
-version = "0.4.0"
+version = "0.5.0"
 authors = ["Zac Burns <That3Percent@gmail.com>"]
 edition = "2018"
 license = "MIT"

